src/promptflow/promptflow/_sdk/data/visualize.j2

```diff
@@ -10,6 +10,6 @@
     <script>
       window.bulk_test_details_data = {{ data }}
     </script>
-    <script src="{{ js_link }}"></script>
+    <script src="./bulkTestDetails.min.js"></script>
```

Comment:
should use const `VIS_JS_BUNDLE_FILENAME` ?

Reply:
good catch, I forget to use that constant during template render

Reply:
updated to still use const during render, a minor update to rename to `js_path`